### PR TITLE
Add /info endpoint to public API server

### PR DESCRIPTION
**Pull request by qibobo**
[#152745861]

### CODE REVIEW

File: api/lib/routes/info.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+module.exports = function(settings) {
+  var path = require('path');
+  var fs = require('fs');
+  var express = require("express");
+  var HttpStatus = require("http-status-codes");
+
+  var router = express.Router();
+  var infoFilePath = path.resolve(settings.infoFilePath);
+  var catalog = JSON.parse(fs.readFileSync(infoFilePath, 'utf8'));
```
Comment (cdlliuy) [inline api/lib/routes/info.js]: why use catalog here?

File: api/lib/routes/info.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+  router.get("/info", function(req, resp) {
+  	resp.status(HttpStatus.OK).json(catalog);
```
Comment (cdlliuy) [inline api/lib/routes/info.js]: same as above

File: api/test/unit/routes/info.test.js [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+          .get("/v1/info")
+          .end(function(error, result) {
+            expect(error).to.equal(null);
+            console.log(result.body)
+            expect(result.statusCode).to.equal(200);
+            expect(result.body).to.deep.equal({
+              "name": "autoscaler",
+              "build": "001",
+              "support": "https://github.com/cloudfoundry-incubator/app-autoscaler",
+              "description": "Automatically increase or decrease the number of application instances based on a policy you define."
```
Comment (cdlliuy) [inline api/test/unit/routes/info.test.js]: could we address the test with a more general way?  using current style, this test case need to be updated to catch up each changes in info.json

File: api/test/unit/config/settings_test.js [hunk 4]
```diff
@@ -12,6 +12,7 @@ describe('config setting Test Suite', function() {
     defaultConfigTemplate = {
       "port": 8080,
       "publicPort": 8081,
+      "infoFilePath": "../api/config/info.json",
```
Comment (cdlliuy) [inline api/test/unit/config/settings_test.js]: missing test to validate "infoFilePath"

**Comment by boyang9527** [comment]
LGTM

**Comment by cdlliuy** [comment]
LGTM

**Pull request merged**